start_bs.py

```diff
@@ -169,13 +171,13 @@ class SampleXYZ(Device):
     from mxtools.raster_flyer import MXRasterFlyer
     raster_flyer = MXRasterFlyer(vector, zebra, detector)
 
-    from nyxtools.robot import DensoOphydRobot
-    from denso_robot import DensoRobot
-    denso_ophyd_robot = DensoOphydRobot("XF:19IDC-ES{Rbt:1}", name="robot")
-    robot = DensoRobot(denso_ophyd_robot) # DensoRobot is the robot_lib API-compatible object
+    from nyxtools.isara_robot import IsaraRobotDevice
+    from denso_robot import OphydRobot
+    ophyd_robot = IsaraRobotDevice("XF19IDC-ES{Rbt:1}", name="robot")
+    robot = OphydRobot(ophyd_robot) # OphydRobot is the robot_lib API-compatible object
     govs = _make_governors("XF:19IDC-ES", name="govs")
     gov_robot = govs.gov.Robot
-
+    robot.canExchangeSamples = True # necessary for us to identify double-gripper availability
```

Comment:
the canExchangeSamples variable is not defined anywhere

Reply:
taken care of with 671207e